journal/src/main/java/io/camunda/zeebe/journal/Journal.java

```diff
@@ -17,30 +17,30 @@
 
 import io.camunda.zeebe.journal.JournalException.InvalidChecksum;
 import io.camunda.zeebe.journal.JournalException.InvalidIndex;
-import org.agrona.DirectBuffer;
 
 public interface Journal extends AutoCloseable {
```

Comment:
I went for reducing the number of overloaded append methods here as all the ones that accepted DirectBuffers were used by tests exclusively. This comes at the cost of a bit more boilerplate in the tests but actually makes sure the exact methods used by production code are tested.